flytekit/annotated/sample.py

```diff
@@ -1,13 +1,14 @@
 from flytekit.annotated.stuff import task, workflow, WorkflowOutputs
+from flytekit.annotated.type_engine import outputs
 
 
-@task(outputs=['s_out'])
-def x(s: int) -> int:
+@task
+def x(s: int) -> outputs(s_out=int):
```

Comment:
tbh I do not like this, because this needs the user to change the interface of their tasks :(

Reply:
Can you elaborate? Change the interface of current flytekit style tasks? Or of py3 style functions?
This is, from python perspective, equivalent to `(int, str)`

Reply:
And if you return a single output, you don't need to do this and we will automatically name it just "output"

Reply:
What I mean, is that they have to have an explicit dependency on 
```python
from flytekit. ... import outputs
#then use outputs
```

What I want is ideally just annotate with one thing and you are done, if you remove the annotation, your code works as is?
Maybe unreasonable